NewForce-Cohort-6/tabloidmvc-the-caramel-apple-pops
# Ja createacategory ticket10

Description
Trello Ticket: [10 - Create A Category](https://trello.com/c/BirbRTep/10-10-create-a-category)
Created code to allow a logged in user to Create a new Category.  
Also corrected the spelling typo on the word "Availble" in the `<h1>` of the category management page.  STYLING CHANGE ONLY
Created the following files and methods:
Create (GET) and Create (POST) methods added in CategoryController.cs
File for Create in Views>Category>Create.cshtml
Added Section Add in CategoryRepository.cs

Type of change
 New feature (non-breaking change which adds functionality)
  
Testing Instructions
git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout <ja-createacategory-ticket10>
If you don't see the most recent version of my files, git pull origin <ja-createacategory-ticket10>
If you haven't created the Tabloid MVC database, please do so.
Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-caramel-apple-pops/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-caramel-apple-pops/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
Click the execute with debugger solid green button to launch the app
In browser, please go to Login on the nav bar and log in with this email: admin@example.com
This should redirect you back to the home page.
Click on Category Management in NavBar
View the list of available Categories (categories should appear in alphabetical order)
When a user is on the Category list page
They select the `Create Category` button
Then they should be directed to a form in which they can enter a new category name
Once an admin has entered a Category name
They click the `Save` button
Then a new category should be saved to the database
and the admin should be directed to the Category list page
The new category should appear on the list page
 **NOTE:** For the time being it is acceptable to treat all users as `admin` users. There is a future story about enforcing user permissions.


 Checklist:
